__tests__/feature/editing/editingLiteral.test.js

```diff
@@ -41,7 +41,6 @@ describe('editing a literal property', () => {
     fireEvent.click(editBtn)
     expect(input).not.toBeDisabled()
     expect(input).toHaveValue('foo')
-    expect(screen.queryAllByText('foo').length).toBeFalsy()
```

Comment:
Removed, apparently because this passes with `<input>`-based literal fields but fails with `<textarea>`-based ones.

Reply:
Can you change it to: `expect(screen.queryAllByText('foo').length).toBeLessThan(2)`?

Reply:
Apparently the `textarea` is also counted as a text value. This is supposed to test that the little rbt-token text of the input value has dissappeared.

Reply:
@jgreben :speech_balloon: 
> Apparently the `textarea` is also counted as a text value. This is supposed to test that the little rbt-token text of the input value has dissappeared.

Done! Thanks for the suggestion.